### DISCUSSION TITLE
How do I change the email associated with my user?

### DISCUSSION THREAD

**Discussion by yorinasub17** [gruntwork-io/knowledge-base#231] [Help] [answered]
I want to update just my email in app.gruntwork.io, while keeping everything else (github user id, etc). How can I do that?

**Comment by oredavids** [comment, marked as answer]
Currently, app.gruntwork.io does not support changing emails.

A workaround, in order to retain GitHub access **and** not lose forks(if you have one), is to do the following in this **exact order**:

1. Have the new email added to your Organization in _app.gruntwork.io_ by your Admin (this will result in an email invitation)
2. Accept the invitation for the new email to create a new user and sign in
3. Link your current Github account to the new user account
4. Have the Admin remove the old user account from your Organization in _app.gruntwork.io_ to prevent duplicate license usage
